### PR TITLE
Search: try adding chart to wp-admin

### PR DESCRIPTION
**Pull request by bluefuton**
#### Changes proposed in this Pull Request:
Try adding a chart.js bar chart to wp-admin.

#### Does this pull request change what data or activity we track or use?
No.

#### Testing instructions:
TBC.

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -754,6 +751,8 @@ importers:
       '@wordpress/element': 4.1.0
       '@wordpress/i18n': 4.3.0
       '@wordpress/icons': 6.2.0
+      chart.js: ^3.5.1
```
Comment (bluefuton) [inline pnpm-lock.yaml:754]: Have we added two different packages for chart JS? Do we know which is the right one?

File: projects/packages/search/src/dashboard/components/record-meter/index.jsx [hunk 2]
```diff
@@ -13,6 +13,9 @@ import { useSelect } from '@wordpress/data';
  * Internal dependencies
  */
 import { STORE_ID } from 'store';
+import { BarChart } from './barchart.jsx';
```
Comment (bluefuton) [inline projects/packages/search/src/dashboard/components/record-meter/index.jsx:16]: We don't need to include file extensions on these imports.

File: projects/packages/search/src/dashboard/components/record-meter/index.jsx [hunk 3]
```diff
@@ -24,6 +27,12 @@ import './style.scss';
 export default function RecordMeter() {
 	const tierMaximumRecords = useSelect( select => select( STORE_ID ).getTierMaximumRecords() );
 
+	// this.state = {
+	// 	recordInfo: getRecordInfo(createData().data, createData().planInfo),
+	//   };
+
+	const [ state ] = useState( getRecordInfo( createData().data, createData().planInfo ) );
```
Comment (bluefuton) [inline projects/packages/search/src/dashboard/components/record-meter/index.jsx:34]: I _think_ what we need here is something like:

```
const [ recordInfo, setRecordInfo ] = useState( getRecordInfo( createData().data, createData().planInfo ) );
```

...and then you should be access the state using the `recordInfo` variable.

File: projects/packages/search/src/dashboard/components/record-meter/index.jsx (hunk 2 above)
Comment (annacmc) [reply projects/packages/search/src/dashboard/components/record-meter/index.jsx]: ah, right - thank you! 👍

File: projects/packages/search/src/dashboard/components/record-meter/index.jsx (hunk 3 above)
Comment (annacmc) [reply projects/packages/search/src/dashboard/components/record-meter/index.jsx]: Yep, this works nicely!

**Comment by bluefuton** [comment]
Bit hard to rescue this after the rebase. We'll spin up a new PR.

**Pull request closed without merging**